### PR TITLE
Revising PSK generation

### PR DESCRIPTION
**Pull request by jr0d**
This is a second attempt at updating dcos-ansible to support PSK generation for Exhibitor TLS bootstrap.

### CODE REVIEW

**Comment by jr0d** [comment]
Tested with universal installer:

```
$ ansible -i inventory all -m shell -a "stat /var/lib/dcos/.dcos-bootstrap-ca-psk" -u centos
54.245.219.12 | CHANGED | rc=0 >>
  File: ‘/var/lib/dcos/.dcos-bootstrap-ca-psk’
  Size: 16        	Blocks: 8          IO Block: 4096   regular file
Device: ca01h/51713d	Inode: 67109058    Links: 1
Access: (0600/-rw-------)  Uid: (    0/    root)   Gid: (    0/    root)
Context: system_u:object_r:var_lib_t:s0
Access: 2019-09-05 15:08:17.767303426 +0000
Modify: 2019-09-05 15:08:17.647306302 +0000
Change: 2019-09-05 15:08:17.771303330 +0000
 Birth: -

54.202.39.67 | CHANGED | rc=0 >>
  File: ‘/var/lib/dcos/.dcos-bootstrap-ca-psk’
  Size: 16        	Blocks: 8          IO Block: 4096   regular file
Device: ca01h/51713d	Inode: 67109058    Links: 1
Access: (0600/-rw-------)  Uid: (    0/    root)   Gid: (    0/    root)
Context: system_u:object_r:var_lib_t:s0
Access: 2019-09-05 15:04:21.821048800 +0000
Modify: 2019-09-05 15:04:21.703054371 +0000
Change: 2019-09-05 15:04:21.826048564 +0000
 Birth: -

34.216.251.149 | CHANGED | rc=0 >>
  File: ‘/var/lib/dcos/.dcos-bootstrap-ca-psk’
  Size: 16        	Blocks: 8          IO Block: 4096   regular file
Device: ca01h/51713d	Inode: 67109058    Links: 1
Access: (0600/-rw-------)  Uid: (    0/    root)   Gid: (    0/    root)
Context: system_u:object_r:var_lib_t:s0
Access: 2019-09-05 15:12:11.372351217 +0000
Modify: 2019-09-05 15:12:11.248352062 +0000
Change: 2019-09-05 15:12:11.376351190 +0000
 Birth: -
```

File: roles/dcos_bootstrap/tasks/main.yml [hunk 1]
```diff
@@ -74,6 +74,56 @@
 - name: Create install directory/genconf
   file: path={{ download_path }}/genconf state=directory mode=0755
 
+
```
Comment (sebbrandt87) [inline roles/dcos_bootstrap/tasks/main.yml]: extra empty line?

Comment (sebbrandt87) [inline roles/dcos_bootstrap/tasks/main.yml]: extra empty line?

File: roles/dcos_bootstrap/tasks/main.yml [hunk 2]
```diff
@@ -74,6 +74,56 @@
...
+- name: Delegate psk variable from password generator
+  set_fact:
+    delegated_psk: "{{ random_psk_result }}"
+  delegate_to: "{{ item }}"
+  delegate_facts: true
+  with_items: "{{ groups['masters'] }}"
+  when: dcos.enterprise_dcos and not psk_stat.stat.exists
+
+
```
Comment (sebbrandt87) [inline roles/dcos_bootstrap/tasks/main.yml]: extra empty line?

**Review by sebbrandt87 — CHANGES_REQUESTED** [review]
looking good, we would add tomorrow dcos-ansible enterprise test run to integrate that always running test also in CI

File: roles/dcos_bootstrap/tasks/main.yml (hunk 1 above)
Comment (jr0d) [reply roles/dcos_bootstrap/tasks/main.yml]: not sure how those snuck in. fixing

**Comment by sebbrandt87** [comment]
test against enterprise looking good
```TASK [dcos_bootstrap : Set PSK path] *******************************************
    ok: [bootstrap1-centos7-enterprise-ci-mesos-jenkins-58d4b86021814e4b97865f9474daf0dd-py36]
    
    TASK [dcos_bootstrap : Stat the existing PSK] **********************************
    ok: [bootstrap1-centos7-enterprise-ci-mesos-jenkins-58d4b86021814e4b97865f9474daf0dd-py36]
    
    TASK [dcos_bootstrap : Read existing PSK contents] *****************************
    ok: [bootstrap1-centos7-enterprise-ci-mesos-jenkins-58d4b86021814e4b97865f9474daf0dd-py36]
    
    TASK [dcos_bootstrap : Generate PSK for exhibitor bootstrap] *******************
    skipping: [bootstrap1-centos7-enterprise-ci-mesos-jenkins-58d4b86021814e4b97865f9474daf0dd-py36]
    
    TASK [dcos_bootstrap : Write CA service PSK] ***********************************
    skipping: [bootstrap1-centos7-enterprise-ci-mesos-jenkins-58d4b86021814e4b97865f9474daf0dd-py36]
    
    TASK [dcos_bootstrap : Delegate psk variable from disk] ************************
    ok: [bootstrap1-centos7-enterprise-ci-mesos-jenkins-58d4b86021814e4b97865f9474daf0dd-py36 -> 3.226.241.182] => (item=master1-centos7-enterprise-ci-mesos-jenkins-58d4b86021814e4b97865f9474daf0dd-py36)
    
    TASK [dcos_bootstrap : Delegate psk variable from password generator] **********
    skipping: [bootstrap1-centos7-enterprise-ci-mesos-jenkins-58d4b86021814e4b97865f9474daf0dd-py36] => (item=master1-centos7-enterprise-ci-mesos-jenkins-58d4b86021814e4b97865f9474daf0dd-py36) 

...

TASK [dcos_master : Create /var/lib/dcos] **************************************
    ok: [master1-centos7-enterprise-ci-mesos-jenkins-58d4b86021814e4b97865f9474daf0dd-py36]

TASK [dcos_master : Write CA service PSK] **************************************
    ok: [master1-centos7-enterprise-ci-mesos-jenkins-58d4b86021814e4b97865f9474daf0dd-py36]
```

**Review by sebbrandt87 — APPROVED** [review]

**Pull request merged**